Lab1.ipynb

```diff
@@ -1,6 +1,822 @@
+      "text/plain": [
+       "<IPython.core.display.Image object>"
+      ]
+     },
+     "execution_count": 11,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# 3. C.\n",
+    "from IPython.display import Image\n",
+    "Image(filename='lab1-3C.png')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 12,
+   "id": "b917ef02-a630-4f89-82ca-4825311b4736",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "-0.34657359027997275"
+      ]
+     },
+     "execution_count": 12,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# 3. D.\n",
+    "sigma1 = stats.norm.ppf(1-stats.lognorm.cdf(val, s), loc = 0, scale = 0.25)\n",
+    "sigma1"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 13,
+   "id": "3926efc1-1cdb-4f45-a560-0ef968a35635",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# The result is negative because when we convert it to Gaussian\n",
+    "# distribution, the resulting sigma is less than the mean of Gaussian."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 14,
+   "id": "635e716f-edf7-44b8-b606-891017f1a12e",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "-0.6931471805599447"
+      ]
+     },
+     "execution_count": 14,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# 4. \n",
+    "# We change the hypothetical value to 4 and keep the value of s at 0.5\n",
+    "val = 4\n",
+    "# What is the chance to observe the given data at 4 due to background hiss?\n",
+    "sigma2 = stats.norm.ppf(1-stats.lognorm.cdf(val, s), loc = 0, scale = 0.25)\n",
+    "sigma2"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 15,
+   "id": "23016e35-e2c0-41db-ae7a-839911dc7032",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "0.0"
+      ]
+     },
+     "execution_count": 15,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# 4. cont.\n",
+    "# We change the hypothetical value to 1 and keep the value of s at 0.5\n",
+    "val = 1\n",
+    "# What is the chance to observe the given data at 1 due to background hiss?\n",
+    "sigma3 = stats.norm.ppf(1-stats.lognorm.cdf(val, s), loc = 0, scale = 0.25)\n",
+    "sigma3"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 16,
+   "id": "370063bb-d917-4266-8fd0-d4736ef4f6f0",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "0.34657359027997275"
+      ]
+     },
+     "execution_count": 16,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ]
```

Comment:
I think you can also mention the reason for having discrete probabilities and sigmas for discrete distribution. What's the biggest takeaway from having discrete distribution in experiments ?